src/LaravelDebugbar.php

```diff
@@ -606,6 +606,43 @@ protected function isJsonRequest($request)
     }
 
     /**
+     * Check if requested route is not blacklisted.
+     *
+     * @param  \Symfony\Component\HttpFoundation\Request $request
+     * @return bool
+     */
+    protected function isAllowedRoute($request)
+    {
+        $ignored_routes = $this->app['config']->get('debugbar.ignored_routes');
+        $current_route = $request->getRequestUri();
+
+        if($current_route[strlen($current_route)-1] == "/")
+            $current_route = substr($current_route, 0, strlen($current_route)-1);
+
+        if($current_route == "")
+            $current_route = "/";
+
+        foreach($ignored_routes as $ignored_route)
+        {
+            if($ignored_route == "")
+                $ignored_route = "/";
+
+            $ignored_route = preg_replace("({[^{|}]+})", "[A-Za-z0-9]+", $ignored_route);
+
+            if(strlen($ignored_route) > 1 && $ignored_route[0] != "/") // To allow rules writes as "/route" and "route"
+                $ignored_route = "/".$ignored_route;
+
+            if($ignored_route[strlen($ignored_route)-1] == "/")
```

Comment:
I made sure to use correct formatting and then used wrong one... Fortunately it can be easily fixed.